src/hybrid/hybrid_request.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "aggregate/aggregate.h"
+#include "pipeline/pipeline.h"
+#include "hybrid/hybrid_scoring.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct {
+    arrayof(AREQ*) requests;
+    size_t nrequests;
+    QueryError tailError;
+    QueryError *errors;
+    Pipeline tail;
```

Comment:
* Consider having similar naming to the `AREQ`.
* Also, we don't have the error (`tailError`) in the `AREQ` consider aligning that as well (not super important, I'm fine with the error here as well).
* Please add documentation for the fields